### PR TITLE
New sample app: RequestActivity with "Create Ticket" menu button.

### PR DESCRIPTION
**Pull request by brendan-fahy**
## Changes
- Add new sample app, subclass of `RequestActivity` with added menu, with button to launch `ContactZendeskActivity`.

## Reviewers
@baz8080 @schlan @ndobri @a1cooke 

## FYI
@mathewcropper 

## Risks
None.

### CODE REVIEW

File: request_list_with_add_ticket_sample_app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,33 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 23
```
Comment (schlan) [inline request_list_with_add_ticket_sample_app/build.gradle]: I think we should extract all of those constants (compileSdk, support, sdk, ...), to make sure all sample apps using the same version and to keep maintenance effort low.

**Comment by brendan-fahy** [comment]
How do you like them extracted constants/apples, @schlan?

**Comment by baz8080** [comment]
I like those 🍏 🍎

**Comment by baz8080** [comment]
The current exchange of apples to plus ones is 👍

**Pull request merged**